repose-aggregator/tests/release-verification/src/vagrant/deb/Vagrantfile

```diff
@@ -0,0 +1,172 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+repose_version = ENV['REPOSE_VERSION'] || 'local'
+
+$update_yum = <<SCRIPT
+echo "-------------------------------------------------------------------------------------------------------------------"
+echo "Updating Apt packages"
+echo "-------------------------------------------------------------------------------------------------------------------"
+
+# update package lists from the server
+apt-get update -qq
+#apt-get upgrade -yq
```

Comment:
I don't think I did? I just added another "q" to the update. I believe the upgrade line was commented out in the original vagrant branch.